paddle/fluid/sub_graph/sub_graph_checker.cc

```diff
@@ -0,0 +1,419 @@
+// Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "paddle/fluid/sub_graph/sub_graph_checker.h"
```

Comment:
```suggestion
#include "paddle/fluid/sub_graph/checker.h"
```
这样感觉更简洁些？因为目录名已经有了sub_graph